e2e/plugin-js-packages-e2e/mocks/fixtures/npm-repo/code-pushup.config.ts

```diff
@@ -0,0 +1,32 @@
+import jsPackagesPlugin from '@code-pushup/js-packages-plugin';
+import type { CoreConfig } from '@code-pushup/models';
+
+export default {
+  persist: {
+    outputDir: './',
+    filename: 'report',
+    format: ['json'],
+  },
+  plugins: [await jsPackagesPlugin({ packageManager: 'npm' })],
+  categories: [
+    {
+      slug: 'security',
+      title: 'Security',
+      refs: [
+        { type: 'group', plugin: 'js-packages', slug: 'npm-audit', weight: 1 },
+      ],
+    },
+    {
+      slug: 'updates',
+      title: 'Updates',
+      refs: [
+        {
+          type: 'group',
+          plugin: 'js-packages',
+          slug: 'npm-outdated',
+          weight: 1,
+        },
+      ],
+    },
+  ],
```

Comment:
I don't think the categories need to be included here. The test only asserts the slugs are in the resulting report, but that's not the responsibility of this plugin.